### PR TITLE
Fix broken jail

### PR DESCRIPTION
**Pull request by dramich**
Problem:
Jailed did not handle multi-threaded

Solution:
Lock the jailer runs

### CODE REVIEW

File: package/jailer.sh [hunk 1]
```diff
@@ -24,6 +24,10 @@ cp -r /lib64 /opt/jail/$NAME
 cp /etc/ssl/certs/ca-certificates.crt /opt/jail/$NAME/etc/ssl/certs
 cp /etc/resolv.conf /opt/jail/$NAME/etc/
 
+if [ -d /var/lib/rancher/management-state/bin ]; then
+    cp -r /var/lib/rancher/management-state/bin /opt/drivers/management-state
```
Comment (daxmc99) [inline package/jailer.sh]: Why do we need this?

Comment (dramich) [reply package/jailer.sh]: Upgrade path, `/var/lib/rancher/management-state/bin` is where node drivers live in old rancher, on upgrade they need to be moved to our `new` install location to be pulled into the jail

**Comment by daxmc99** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**